pages/portfolio.html

```diff
@@ -4,9 +4,37 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <link href="styles/style.css" rel="stylesheet">
+  <title>Portfolio</title>
 </head>
 <body>
-  
+  <center>
+  <h1>Portfolio</h1>
+  <p>
+    While a student in Ada Developer's Academy, I have worked on a variety of projects. Here are some of my favorite ones:
+  </p>
+  <p class="container">
+  <img src="https://imgur.com/a/v49wllZ" width="100" height="100">
+  <img src="https://imgur.com/a/VjZQqKE" width="100" height="100">
+</p>
+  <ul>
+    <li> 
+      <a href="https://github.com/goblinbrooke/retro-video-store">Retro Video Store</a> - A video store application 
+      that uses a MySQL database to store video information. 
+    </li>
+    <li>
+      <a href="https://github.com/goblinbrooke/task-list-api">Task List API</a> - An API that allows users to create and accomplish
+      tasks and set goals.
+    </li>
+  </center>
```

Comment:
Missing a close `ul`

```suggestion
    </ul>
  </center>
```